packages/modul-components/src/components/maskedfield/maskedfield.ts

```diff
@@ -0,0 +1,77 @@
+import { PluginObject } from 'vue';
+import Component from 'vue-class-component';
+import { Prop } from 'vue-property-decorator';
+import { InputLabel } from '../../mixins/input-label/input-label';
+import { InputManagement } from '../../mixins/input-management/input-management';
+import { InputState } from '../../mixins/input-state/input-state';
+import { InputWidth } from '../../mixins/input-width/input-width';
+import L10nPlugin from '../../utils/l10n/l10n';
+import uuid from '../../utils/uuid/uuid';
+import { ModulVue } from '../../utils/vue/vue';
+import { MASKEDFIELD_NAME } from '../component-names';
+import { InputMaskOptions, MInputMask } from '../input-mask/input-mask';
+import InputStyle from '../input-style/input-style';
+import ValidationMesagePlugin from '../validation-message/validation-message';
+import WithRender from './maskedfield.html?style=./maskedfield.scss';
+
+@WithRender
+@Component({
+    mixins: [
+        InputState,
+        InputWidth,
+        InputLabel,
+        InputManagement
+    ],
+    components: {
+        MInputMask
+    }
+})
+export class MMaskedfield extends ModulVue {
```

Comment:
Moi j'aurais p-e mis directement implements InputMaskOptions et j'aurais ajouter une prop pour chacune des bebelles dans inputMaskOptions.. comme ça on aurait pu faire absolument tout ce qu'on veut avec le field.

Autre alternative: Tu aurais pu mettre inheritAttrs à false et faire 
private get inputMaskOptions(): InputMaskOptions {
        return this.$attrs;
    }

Ce qui nous permet de mettre des attributs pour chaque propriété dans InputMaskOptions sans avoir à définir une props pour.

Reply:
Ou sinon prendre en props un objet options de type InputMaskOptions.  Mais je sais que Charles est pas un fan

Reply:
@Mboulianne c'est pour faire le code postal dans Brio pour adresse. On a décider de mettre minimum au début pour répondre à mon besoin, si d'autre besoin arrive que les props présentement, on ajustement et exposera direct le MInputMaskOption peut-être.